tests/command.positionalOptions.test.js

```diff
@@ -362,13 +362,36 @@ describe('program with action handler and positionalOptions and subcommand', ()
 
 // ------------------------------------------------------------------------------
 
-test('when program not positional and turn on passthrough in subcommand then error', () => {
-  const program = new commander.Command();
-  const sub = program.command('sub');
+describe('illegal passThroughOptions', () => {
+  test('when program not positional and turn on passThroughOptions in subcommand then error', () => {
+    const program = new commander.Command();
+    const sub = program.command('sub');
+
+    expect(() => {
+      sub.passThroughOptions();
+    }).toThrow();
+  });
+
+  test('when program not positional and add subcommand with passThroughOptions then error', () => {
+    const program = new commander.Command();
+    const sub = new commander.Command('sub')
+      .passThroughOptions();
 
-  expect(() => {
-    sub.passThroughOptions();
-  }).toThrow();
+    expect(() => {
+      program.addCommand(sub);
+    }).toThrow();
+  });
+
+  test('when program has subcommand with passThroughOptions and reset to non-positional then error', () => {
```

Comment:
I do not want to bother testing for this. Resetting `enablePositionalOptions` is a very unlikely use case.

Reply:
Hmm, okay, so should I even remove the check from the code?

Reply:
Would need to update the ChangeLog, the change would need to be more specific. It is a bit strange to exclude one particular case in which the inconsistent state is reached…

Reply:
Yes, thanks.

Reply:
Resolved in [9d4c96a](https://github.com/tj/commander.js/pull/1937/commits/9d4c96a4fc40590dced784d24f26661d183fb512).